finesse/gui/stepper_motor_view.py

```diff
@@ -53,5 +65,21 @@ def _preset_clicked(self, btn: QPushButton) -> None:
         # If the motor is already moving, stop it now
         pub.sendMessage(f"device.{STEPPER_MOTOR_TOPIC}.stop")
 
+        pub.sendMessage(f"device.{STEPPER_MOTOR_TOPIC}.notify_on_stopped")
         target = float(self.angle.value()) if btn is self.goto else btn.text().lower()
         pub.sendMessage(f"device.{STEPPER_MOTOR_TOPIC}.move.begin", target=target)
+
+    def _update_mirror_position_display(self, moved_to: float | None) -> None:
+        """Display the angle the mirror has moved to.
+
+        moved_to will be None if mirror is moving, in which case display "Moving...",
+        otherwise, display angle or its associated name.
+        """
+        if moved_to is None:
+            self.mirror_position_display.setText("Moving...")
```

Comment:
This is still something that needs tweaking if we want it to display "moving" when the mirror is moving. You need to have a separate handler which sets the text to moving when you get a `move.begin` message.